DB/api/__init__.py

```diff
@@ -1 +0,0 @@
-__all__ = ["Analysis_3", "query_db", "insert_db","detect_events", "handler", "server", "url_parser"]
```

Comment:
You dont actually need anything in the __init__ files, they are just there to structure the modules/packages